test/memory/sungeun/refCount/arrays.chpl

```diff
@@ -141,5 +144,5 @@ proc main() {
   m2 = memoryUsed();
   writeln("\t", m2-m1, " bytes leaked");
   if printMemStats then printMemAllocs();
-
+  */
```

Comment:
Honestly I'm not sure what to do with these lines. I think Brad is hoping to remove reindexing as an operation that is supported. At one point this test failed when reindexing was enabled but I'm not sure if it would work now. I think my preference would be to leave them commented out, but if that makes sense to you I can at least explain the situation in a comment.